frontend/src/routes/_dashboard/index.lazy.tsx

```diff
@@ -101,83 +80,129 @@ export function ProjectList() {
   const [currentPage, setCurrentPage] = React.useState(1);
   const [perPage, setPerPage] = React.useState(10);
 
+  const query = useProjectList();
+
+  if (query.isLoading) {
+    return <Loader />;
```

Comment:
This should be a skeleton, not a loader.

A skeleton is a kind of loader that shows a placeholder of the content while it is loading, it is usually better than a generic loader because it makes it seems like the content loads faster.

Here are some notable examples :

- my github clone :
<img width="1246" alt="image" src="https://github.com/zane-ops/zane-ops/assets/38298743/42b9f6ca-07f9-484d-bbed-25196b14676e">

- OpenStatus : 
<img width="1317" alt="image" src="https://github.com/zane-ops/zane-ops/assets/38298743/c9939bad-b9c3-4915-a6ce-0f82f8b88005">

- Vercel :
<img width="941" alt="image" src="https://github.com/zane-ops/zane-ops/assets/38298743/088b5a0a-850c-4cd6-8172-48214b8a6cee">

Reply:
I will put the skeleton in a another PR